docs/examples/can_lin_diff.rst

```diff
@@ -0,0 +1,20 @@
+Adapting CAN examples to LIN
+============================
+
+This displays the diff of ``can_frame_stream_io.py`` and 
+``lin_frame_stream_io.py`` to demonstrate the changes required to
+update CAN example code for LIN.
+
+To adapt this example to LIN frames, reference signals in a database that use LIN:
```

Comment:
1. "to adapt the examples from CAN to LIN"...
2. This should go before the diff